plugins/package-managers/cargo/src/main/kotlin/CargoMetadata.kt

```diff
@@ -93,6 +93,31 @@ internal data class CargoMetadata(
         /** The Package ID of this node. */
         val id: String,
         /** The dependencies of this package, an array of Package IDs. */
-        val dependencies: List<String>
+        val dependencies: List<String>,
+
+        /**
+         * The dependencies of this package. This is an alternative to "dependencies" which contains additional
+         * information. In particular, this handles renamed dependencies.
```

Comment:
Sounds to me like `dependencies` would contain additional information, but looking at the datatypes it seems that `deps` contains additional dependencies. Consider replacing "which" with "that".

Reply:
I've taken all of these comments as-is from [here](https://doc.rust-lang.org/cargo/commands/cargo-metadata.html#json-format), so I'd prefer not to change them, or keep changes to a minimum.